docs/site/contest_formats.md

```diff
@@ -37,7 +46,19 @@ For example, say a submission with a score of `50/100` is submitted 23 minutes b
 
 ## AtCoder Contest Format
 
-The `AtCoder` contest format is based on the contest format used by AtCoder. As such, your score equals the sum of your highest-scoring submission on each problem, and ties are broken based on the time of your last score-changing submission plus your penalty.
+The `AtCoder` contest format is based on the contest format used by AtCoder.
+
+The score is equal to the sum of the highest-scoring submission on each problem, and ties are broken based on the time of the last score-changing submission plus the penalty.
 
-The penalty is specified by the `penalty` field, and defaults to 5 minutes.
+The penalty is specified by the `penalty` field, and defaults to `5` minutes.
 The penalty is equal to the total number of incorrect submissions prior to the highest scoring submission on each solved problem, multiplied by the specified value, in minutes, and is added to the cumulative time.
+
+## ICPC Contest Format
+
+The `ICPC` contest format is based on the contest format used by the ICPC.
+
+The score is equal to the number of problems solved, and ties are broken based on the sum of the elapsed time that a correct submission was submitted to each problem plus the penalty.

```

Comment:
There are 3 tiebreakers, third one is the time of last score-alterating submission iirc